src/core/ext/resolver/dns/native/dns_resolver.c

```diff
@@ -71,9 +72,9 @@ typedef struct {
   /** pending next completion, or NULL */
   grpc_closure *next_completion;
   /** target result address for next completion */
-  grpc_resolver_result **target_result;
+  grpc_channel_args **target_result;
```

Comment:
s/_result/_channel_args? or .../_args

Same for other variables of the late `grpc_resolver_result` type.